dependencies.yaml

```diff
@@ -557,14 +560,14 @@ dependencies:
     common:
       - output_types: [conda]
         packages:
-          - &transformers transformers=4.36.2 # newer versions are incompatible with our pinned version of huggingface_hub
+          - &transformers transformers=4.40
           - anyio>=3.7
           - arxiv=1.4
           - httpx>=0.23,<0.28 # work-around for https://github.com/openai/openai-python/issues/1915
-          - huggingface_hub=0.20.2 # work-around for https://github.com/UKPLab/sentence-transformers/issues/1762
+          - huggingface_hub=0.24
```

Comment:
Is this pin for a particular reason? Can we not use a more recent version?

Reply:
hmm.... I'll have to test that